src/main/java/io/cdap/plugin/servicenow/sink/model/ServiceNowSchemaField.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.servicenow.sink.model;
+
+/**
+ * Model class for Schema Field from Schema API
+ */
+public class ServiceNowSchemaField {
+  private String label;
+  private String exampleValue;
+  private String internalType;
+  private String name;
+
+  public String getLabel() {
+    return label;
+  }
+
+  public void setLabel(String label) {
```

Comment:
imilar comment to above, set all fields once in a constructor and don't modify state.

Reply:
All fields are not being set in the constructor. cbfb212c77242fe61b2ef14c690a9079b6a9e22f